File: csharp/Validators/ArgumentLimitValidator.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Validators
{
    public class ArgumentLimitValidator: ArgumentCountValidator
    {
        public ArgumentLimitValidator(List<string> args, ushort count, string message): base(args, count, message)
        {
        }

        public override bool Validate()
        {
            if (Arguments.Count >= Count)
            {
                Console.WriteLine("ERROR: " + Message);
                return false;
            }

            return true;
        }
    }
}

File: python/Validators/ArgumentLimitValidator.py
from .AbstractListValidator import *


class ArgumentLimitValidator(AbstractListValidator):
    def __init__(self, validate_this_list, count, message):
        super().__init__(validate_this_list, count, message)

    def validate(self):
        if self._validate_this_list.__len__() >= self._count:
            print(self._error_preface() + self._message)
            return False
        else:
            return True
